<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitoringWindow</class>
 <widget class="QMainWindow" name="MonitoringWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>729</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>АРМ мониторинга СНА</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../DbUi/DbUi.qrc">
    <normaloff>:/icons/Icons/Analysis.ico</normaloff>:/icons/Icons/Analysis.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidgetMain">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <widget class="QWidget" name="tabState">
       <attribute name="icon">
        <iconset resource="../DbUi/DbUi.qrc">
         <normaloff>:/icons/Icons/TabState.png</normaloff>:/icons/Icons/TabState.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Состояние</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="FormSystem" name="formSystem" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabLog">
       <attribute name="icon">
        <iconset resource="../DbUi/DbUi.qrc">
         <normaloff>:/icons/Icons/TabLog.png</normaloff>:/icons/Icons/TabLog.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Лог состояний</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="FormSysLog" name="formSysLog" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEvent">
       <attribute name="icon">
        <iconset resource="../DbUi/DbUi.qrc">
         <normaloff>:/icons/Icons/TabEvent.png</normaloff>:/icons/Icons/TabEvent.png</iconset>
       </attribute>
       <attribute name="title">
        <string>События</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="FormEvents" name="formEvents" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="menuAutoRefresh">
     <property name="title">
      <string>Период обновления</string>
     </property>
     <addaction name="actionRefreshRate0_5"/>
     <addaction name="actionRefreshRate1"/>
     <addaction name="actionRefreshRate2"/>
     <addaction name="actionRefreshRate5"/>
     <addaction name="actionRefreshRate30"/>
     <addaction name="separator"/>
     <addaction name="actionRefreshPause"/>
    </widget>
    <addaction name="actionRefresh"/>
    <addaction name="menuAutoRefresh"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetPreview">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Предпросмотр</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="labelPreview">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Monitoring.qrc">:/Icons/No image.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../DbUi/DbUi.qrc">
     <normaloff>:/icons/Icons/ToolRefresh.png</normaloff>:/icons/Icons/ToolRefresh.png</iconset>
   </property>
   <property name="text">
    <string>Обновить сейчас</string>
   </property>
   <property name="toolTip">
    <string>Обновить сейчас</string>
   </property>
  </action>
  <action name="actionRefreshRate0_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0.5 секунды</string>
   </property>
  </action>
  <action name="actionRefreshRate1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 секунда</string>
   </property>
  </action>
  <action name="actionRefreshRate2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2 секунды</string>
   </property>
  </action>
  <action name="actionRefreshRate5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5 секунд</string>
   </property>
  </action>
  <action name="actionRefreshRate30">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>30 секунд</string>
   </property>
  </action>
  <action name="actionRefreshPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Пауза</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FormSystem</class>
   <extends>QWidget</extends>
   <header>FormSystem.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormSysLog</class>
   <extends>QWidget</extends>
   <header>FormSysLog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormEvents</class>
   <extends>QWidget</extends>
   <header>FormEvents.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../DbUi/DbUi.qrc"/>
  <include location="Monitoring.qrc"/>
 </resources>
 <connections/>
</ui>
